test/commands-tests/node.js

```diff
@@ -66,9 +66,26 @@ describe('ForgAE Node', () => {
         }
     })
 
-    it('Process should start local compiler', async () => {
+    after(async () => {
+
+        let running = await waitForContainer();
+        if (running) {
+            await execute(constants.cliCommands.NODE, [constants.cliCommandsOptions.STOP], executeOptions)
+        }
+        fs.removeSync(`.${ constants.nodeTestsFolderPath }`)
+    })
+})
+
+describe('ForgAE Node - check if compiler is running too', () => {
+
+    before(async () => {
+        fs.ensureDirSync(`.${ constants.nodeTestsFolderPath }`)
+
+        await execute(constants.cliCommands.INIT, [], executeOptions)
         await execute(constants.cliCommands.NODE, [], executeOptions)
-        await waitForContainer();
+    })
+
+    it('Local compiler should be running.', async () => {
```

Comment:
I think that we should run also `waitForContainer()`here because if the node is not started the test will fail, or I am wrong?